test/asyncapi_test.go

```diff
@@ -59,7 +59,7 @@ func (s *CLITestSuite) TestAsyncApiImport() {
 	}
 }
 
-func (s *CLITestSuite) TestAsyncApiImportWithWorkflow() {
+func (s *CLITestSuite) TestAsyncapiImport_WithWorkflow() {
```

Comment:
AsyncApi?

Reply:
The command is called "asyncapi" and not "async-api", so this follows the pattern.